addons/l10n_gh/__manifest__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Ghana - Accounting',
+    'category': 'Localization/Account Charts',
+    'description': """
+This is the base module to manage the accounting chart for Ghana.
+===========================================================================
+
+* Chart of Accounts.
+
+    """,
+    'author': 'erpSOFTapp',
+    'license': 'AGPL-3',
+    'depends': [
+        'account',
+        'account_accountant',
+                'base'
+
+    ],
+    'data': [
+        'data/l10n_gh_chart_data.xml',
+        'data/account.account.template.csv',
+        'data/account.chart.template.csv',
+        'data/account.tax.group.csv',
+        'data/account_tax_data.xml',
+        'data/res.country.state.csv',
```

Comment:
better in base and only if it is really needed in the postal addresses

Reply:
Yes for Ghana the regions in the states CSV is needed in the postal addresses
When you say better in base can you clarify that aspect?

Reply:
There is a res.country.state.csv file in the base module (odoo/odoo) already.  It is better that everyone has them then.

Reply:
Ok will you take it out of this module and move it to base then?